queries/bibtex/highlights.scm

```diff
@@ -0,0 +1,49 @@
+; CREDITS @pfoerster (taken from https://github.com/latex-lsp/tree-sitter-bibtex)
+
+[
+  (string_type)
+  (preamble_type)
+  (entry_type)
+] @keyword
+
+[
+  (junk)
+  (comment)
+] @comment
+
+[
+  "="
+  "#"
+] @operator
+
+(command) @function.builtin
+
+(number) @number
+
+(field
+  name: (identifier) @variable.builtin)
+
+(token
+  (identifier) @variable.parameter)
```

Comment:
This is `@parameter` for us. Should we add `@variable.parameter` as a comp mapping (needs to be added to hl_map in highlight.lua and contributing).